src/components/table.js

```diff
@@ -222,7 +224,7 @@ export const FlexTable = ({
     role: 'table',
     'aria-rowcount': rowCount + 1, // count the header row too
     'aria-colcount': columns.length,
-    'aria-label': tableName,
+    'aria-label': ariaLabel,
```

Comment:
Could you go through this file and move the aria props to the top?
If you would prefer to do a follow up PR for this to keep the PR noise down, I am fine with that as well.